spec/controllers/users_controller_spec.rb

```diff
@@ -100,7 +100,7 @@
 
       params = { name: 'New value', surname: user.surname, birth_date: user.birth_date }
 
-      get :update, params: { id: user.id, user: params }
+      get :update_profile, params: { id: user.id, user: params }
```

Comment:
You have `post '/profile' => 'users#update_profile'` in routes, here you use `get :update_profile, params: { id: user.id, user: params }` and it's working.... weeeird :D

Reply:
hahah true, but all tests passed 🙈